packages/beta/src/types.ts

```diff
@@ -133,3 +134,360 @@ export interface RelationshipsFilter extends CreatedAndLastUpdatedTimeFilter {
+   */
+  targets: ExternalEntityToMatch[];
+  /**
+   * List of objects of pairs of fromId or fromExternalId and toId or toExternalId, that corresponds to entities in matchFrom and matchTo respectively, that indicates a confirmed match used to train the model. If omitted, an unsupervised model is used.
+   */
+  trueMatches?: ExternalEntityTrueMatch[];
+  externalId?: CogniteExternalId;
+  /**
+   * User defined name of the model.
+   */
+  name?: string;
+  /**
+   * User defined description of the model.
+   */
+  description?: string;
+  /**
+   * List of pairs of fields from the target and source items used to calculate features. All source and target items should have all the source and target fields specified here.
+   */
+  matchFields?: EntityMatchingField[];
+  /**
+   * Defines the combination of features used. The options are:
+   * Simple: Calculates a single cosine-distance similarity score for each of the fields defined in keysFromTo. This is the fastest option.
+   * Bigram: Adds similarity score based on the sequence of the terms.
+   * Frequency-Weighted-Bigram: Calculates a similarity score based on the sequence of the terms, giving higher weights to less commonly occurring tokens.
+   * Bigram-Extra-Tokenizers: Similar to bigram, but able to learn that leading zeros and spaces should be ignored in matching.
+   * Bigram-Combo: Calculates all of the above options, relying on the model to determine the appropriate features to use. This is the slowest option.
+   */
+  featureType?: EntityMatchingFeatureType;
+  /**
+   * The classifier used in the model. Only relevant if there are trueMatches/labeled data and a supervised  model is fitted.
+   */
+  classifier?: EntityMatchingClassifier;
+  /**
+   * If True, replaces missing fields in `sources` or `targets` entities, for fields set in set in `matchFields`, with empty strings. Else, returns an error if there are missing data.
+   */
+  ignoreMissingFields?: boolean;
+}
+
+export interface EntityMatchingCreateResponseBase {
+  id?: CogniteInternalId;
+  externalId?: CogniteExternalId;
+  /**
+   * Defines the combination of features used in the entity matcher model.
+   */
+  featureType?: EntityMatchingFeatureType;
+  /**
+   * Name of the classifier used in the model, "Unsupervised" if unsupervised model.
+   */
+  classifier?: EntityMatchingClassifier;
+  /**
+   * If True, missing fields in sources or targets entities set in matchFields, are replaced with empty strings.
+   */
+  ignoreMissingFields?: boolean;
+  /**
+   * List of pairs of fields from the target and source items used to calculate features. All source and target items should have all the source and target fields specified here.
+   */
+  matchFields?: EntityMatchingField[];
+  /**
+   * The ID of original model, only relevant when the model is a retrained model.
+   */
+  originalId?: CogniteInternalId;
+}
+
+export interface EntityMatchingCreateResponse
+  extends EntityMatchingCreateResponseBase,
```

Comment:
same as the other one. `extends { empty }` not good.
copy `EntityMatchingCreateResponseBase` props in side of brakets.
remove `EntityMatchingCreateResponseBase`